src/constants/productSortOptions.ts

```diff
@@ -10,4 +10,20 @@ export const sortOptions: IOption<IProductSortParams>[] = [
     label: 'Price: High to Low',
     value: { sortAttribute: 'price', sortDirection: 'desc' },
   },
+  {
+    label: 'High rating first',
+    value: { sortAttribute: '', sortDirection: 'desc' }, //not implemented on the backend yet
+  },
+  {
+    label: 'Low rating first',
+    value: { sortAttribute: '', sortDirection: 'asc' }, //not implemented on the backend yet
+  },
+  {
+    label: 'Best Sellers',
+    value: { sortAttribute: '', sortDirection: 'asc' }, //not implemented on the backend yet
+  },
 ]
+
+export const defaultSortOption = sortOptions.filter(
```

Comment:
Since this is a function, it would be better to use a verb in its name (for example, `getDefaultSortOption`). As it is not a constant, I suggest moving this function to the component folder `src/components/Product/ProductList`.

Reply:
fixed